src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -3601,7 +3659,7 @@ public String setConfigDir(String dir) {
   static public ServiceInterface loadAndStart(String name, String type) {
     ServiceInterface s = null;
     try {
-      s = create(name, type);
+      s = create(null, name, type);

```

Comment:
dont' pass null as the first arg.. use create(name, type) ...

Reply:
this commit - peers needed to send in their plan
removing this ..
there is only "one" plan